parm/hafs_regional_da_C192s1n4.conf

```diff
@@ -0,0 +1,93 @@
+# Note: These are just example settings. Of course, when model resolution
+# changes, one should also modify the model time step and other related
+# dynamics and physics options, so that they work fine for the desired model
+# resolution.
+[config]
+# Specify the domain center Latitude and Longitude
+domlat=22.0
+domlon=-62.0
+# Same as domlat and domlon but vitals formatted
+vitalsdomlat=220N
+vitalsdomlon=620W
+
+# HAFS launcher requires vitals and a storm.
+# This is a default minimum vitals file for a fake storm.
+# The format of the missing value fields was based on the log
+# output returned when running the launcher after the vitals have
+# been "cleaned" up. That is, if  you enter the fields as all -999 values,
+# the "cleaned" output is returned. In essence I'm treating the vitals
+# module as a factory in the sense that it knows the correct format.
+# NHC 00L FAKE 20120710 0000 250N 0800W -99 -99 -999 -999 -099 -9 -99 -999 -999 -999 -999 M
+# fakestormid is a config option set in the launcher and is the
+# last storm id in the list of storms.
+fakestormid=00L
+fakestormname=NATL
+fakestormorg=NHC
+fakestorm_vitals={fakestormorg} {fakestormid} {fakestormname} {YMD} {HH}{min} {vitalsdomlat} {vitalsdomlon} -99 -99 -999 -999 -099 -9 -99 -999 -999 -999 -999 M
+
+[dir]
+PARMforecast={PARMhafs}/forecast/regional
+
+[grid]
+CASE=C192             ;; FV3 resolution
+LEVS=65              ;; Model vertical levels: 65
+gtype=regional       ;; grid type: uniform, stretch, nest, or regional
+# If gridfixdir is provided and the dir exists, then use the pre-generated static grid fix files under gridfixdir
+#gridfixdir={FIXhafs}/fix_fv3/{CASE}
+gridfixdir=/let/hafs_grid/generate/grid
+# Otherwise, will generate the model grid according to the following grid parameters
+# Need for grid types: stretch, nest and regional
+stretch_fac=1.0001   ;; Stretching factor for the grid
+target_lon={domlon}  ;; center longitude of the highest resolution tile
+target_lat={domlat}  ;; center latitude of the highest resolution tile
+# Need for grid types: nest and regional
+refine_ratio=4       ;; Specify the refinement ratio for nest grid
+istart_nest=12
+jstart_nest=42
+iend_nest=371
+jend_nest=341
+
+[grid_ens]
+CASE_ENS=C192            ;; FV3 resolution
+LEVS_ENS=65              ;; Model vertical levels: 65
+gtype_ens=regional       ;; grid type: uniform, stretch, nest, or regional
+gridfixdir_ens=/let/hafs_grid/generate/grid_ens
+stretch_fac_ens=1.0001   ;; Stretching factor for the grid
+target_lon_ens={domlon}  ;; center longitude of the highest resolution tile
+target_lat_ens={domlat}  ;; center latitude of the highest resolution tile
+refine_ratio_ens=4       ;; Specify the refinement ratio for nest grid
+istart_nest_ens=12
+jstart_nest_ens=42
+iend_nest_ens=371
+jend_nest_ens=341
+
+[forecast]
+ccpp_suite_regional=FV3_HAFS_v0_gfdlmp_tedmf
+
+restart_interval="3 4 5 6 7 8 9"  ;; restart interval in hours for the forecast
```

Comment:
Currently, restart_interval="3 6 9" should be good enough. Not sure we actually output restart files every hour at this point.

Reply:
Yes, thanks!